mercury/static/mercury/style.css

```diff
@@ -439,13 +439,13 @@ input#id_can_msg{
   }
 
   a {
```

Comment:
I generally would avoid changing the style for hyperlinks on the whole site since if this will really be used later, will probably need to be undone. Can you add a CSS class to the divs and then use something like `a .theclass` in this file?

Reply:
yes, I agree it is generally not a good practice. However, I think for our website, it should be fine since we have a few hyperlinks (and we want them to be in violet)